plugins/inputs/snmp_trap/snmp_trap_test.go

```diff
@@ -1357,14 +1357,14 @@ func TestGosmiSingleMib(t *testing.T) {
 			"snmp_trap", // name
 			map[string]string{ // tags
 				"oid":       ".1.3.6.1.6.3.1.1.5.1",
-				"name":      "coldStart",
+				"name":      "coldStart.1.1.5.1",
```

Comment:
This doesn't make much sense. OID `.1.3.6.1.6.3.1.1.5.1` resolves to either `snmpModules.1.1.5.1` or `coldStart`.